modularizar.py

```diff
@@ -0,0 +1,61 @@
+import os
+import datetime #informa ano atual - ATP ETAPA2
+
+#mostrar e somente mostrar o nome ao usuario 
+
+print("Seja bem-vindo a loja de coelhos, entre e escolha um Rodiney Branta")
+
+Cargo_Atual = str(input("Para fazermos sua analise de credito, precisamos que voce entre com algumas informacoes:\nCargo atual:"))
+Salario_Atual = float(input("Salario Atual:"))
+Ano_Nascimento = int(input("Ano de nascimento:"))
+Ano_Atual = datetime.datetime.today() #ATP_ETAPA2
+Idade_Aproximada = (Ano_Atual.year - Ano_Nascimento) #ATP_ETAPA2
+Credito_Atual = ((Salario_Atual*(Idade_Aproximada / 1000)) +100) #ATP_ETAPA2
+meuNomeCompleto = "Rodiney Branta"
+
+os.system("cls")
+
+print(f"Seu cargo atual: {Cargo_Atual} \nSeu Salario atual: {Salario_Atual} \nSeu ano de nascimento: {Ano_Nascimento}")
```

Comment:
Eu não vejo o que Maria leva nesta concatenação em uma string só. Se é para fins acadêmicos, então é perfeitamente aceitável, mas no dia-a-dia da programação, ia levar na mão com uma palmatória 🤣.

O motivo é o mesmo de meu comentário anterior: isole as responsabilidades sempre. 

```suggestion
print(f"Seu cargo atual: {Cargo_Atual}")
print(f"Seu Salario atual: {Salario_Atual}")
print(f"Seu ano de nascimento: {Ano_Nascimento}")
```

Reply:
> print(f"Seu cargo atual: {Cargo_Atual}")
> print(f"Seu Salario atual: {Salario_Atual}")
> print(f"Seu ano de nascimento: {Ano_Nascimento}")

Implementadas as sugestões, a princípio, utilizei para verificar como funcionava o `print()` com as quebras de linha.